apps/tyler-site/app/page.tsx

```
@@ -1,42 +1,73 @@
+            </div>
           </div>
         </div>
-      </div>
-    </main>
+      </main>
+    )
+  }
+
+  return (
+    <>
+      <div dangerouslySetInnerHTML={{ __html: page.html }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)